azure-kusto-ingest/azure/kusto/ingest/ingest_client.py

```diff
@@ -126,30 +126,28 @@ def ingest_from_blob(self, blob_descriptor: BlobDescriptor, ingestion_properties
         queue_client = queue_service.get_queue_client(queue=random_queue.object_name, message_encode_policy=TextBase64EncodePolicy())
         queue_client.send_message(content=content)
 
-    def validate_endpoint_service_type(self):
+    def __validate_endpoint_service_type(self):
```

Comment:
did you mean to make this private (one underscore) or mangled (two underscores)?

Reply:
I did, thanks Yochai!

<aside>You know, for a language that prides itself in being *explicit* and *simple*, a system of underscores to denote 4 unique scenarios seems a bit implicit and complex.</aside>